voice/bxml/verbs/transfer.md

```diff
@@ -32,11 +32,12 @@ Note that the [TransferComplete](../callbacks/transferComplete.md) callback is o
 ### Nested Tags
 Between 1 and 8 `<PhoneNumber>` tags must be nested within the `<Transfer>` tag to define the called parties.  If more than
 one `<PhoneNumber>` is specified, called parties will ring simultaneously and the first to answer will be bridged
-to the original call:
+to the original call. Only one `<SipUri>` tag (without any `<PhoneNumber>`s) may be nested within a `<Transfer>`.
```

Comment:
Right above it says it `must` have 1~8 PhoneNumbers, but that is not true when `SipUri` is used.
Maybe it could be something less restrictive, like `Up to 8 PhoneNumbers can be nested...`.